src/lib/radio/radio.ts

```diff
@@ -149,6 +152,18 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
     }
   }
 
+  @Input()
+  get required(){
+    return this._required;
+  }
+
+  set required(required: boolean) {
+    this._required = required;
+
+    if (this._radios) {
+      this._radios.forEach(radio => radio.required = this.required);
+    }
```

Comment:
@jelbourn I think there is a strange situation here and I'm not good enough in angular 2 yet in order to understand. So if I add an item with `required="true"` then setting the group required property will not affect this one. It's a bit strange behaviour.
I see that for the disabled attribute works the same way and I can see the reason that for disabled you might want to update the disabled radio button directly not through the group, but for required it's enough to have one of the buttons required and then message will show up if you submit. 
Not sure what to do here. Also why is that for disabled there will be an `ng-reflect-disabled` and for required is not. Is this a native behaviour? Thanks